python/simulation.py

```diff
@@ -1706,7 +1706,7 @@ def get_array(self, component=None, vol=None, center=None, size=None, cmplx=None
         else:
             v = self._volume_from_kwargs(vol, center, size)
 
-        _, dirs = mp._get_array_slice_dimensions(self.fields, v, dim_sizes, False)
+        _, dirs = mp._get_array_slice_dimensions(self.fields, v, dim_sizes, False, True)
```

Comment:
This won't work. `mp._get_array_slice_dimensions` in `python/meep.i:498` needs to be updated to take the `snap_empty_dims` parameter and pass it through to `fields::get_array_slice_dimensions`.

Reply:
Okay, I think it's fixed now?